### DISCUSSION TITLE
https://blog.eledah.ir/expressions/art

### DISCUSSION THREAD

**Discussion by giscus** [eledah/quartz_blog#24] [Announcements]
# https://blog.eledah.ir/%DA%A9%D9%84%D9%85%D9%87%E2%80%8C%D9%87%D8%A7/%D9%87%D9%86%D8%B1

تا به حال چند بار دریا را دیده‌اید؟ ده بار؟ صد بار؟ قطعاً اشتباه می‌کنید. مطمئنم که تا به حال یک بار هم دریا را ندیده‌اید. دیدن دریا یعنی درک کردن آب، گفتگوی با موج‌ها، لمس کردن نوازش‌های آن، خیره شدن به رقص آب و گوش کردن به نجواهای آن.

https://blog.eledah.ir/%DA%A9%D9%84%D9%85%D9%87%E2%80%8C%D9%87%D8%A7/%D9%87%D9%86%D8%B1

**Comment by hsbitaraf** [comment]
سلام خداقوت.
در کل متن خوبی بود و آموختنی بود ولی چند نکته به ذهنم می رسه؛ در متن نوعی از گسستگی احساس می شد. یک دلیلش اصرار شما بر شکستن متن به پاراگراف های کاملا مشخص است. دلیل دیگرش اصرار شما بر این است که حتما جمله اول خلاصه پاراگراف باشد. به نظرم بهتر است متن طوری نوشته شود که مفاهیم و کلمات و جملات در هم پیچیده شود طوری که به راحتی نتوانیم آن ها را از هم جدا کنیم. ارادتمند شما؛ حسین.
